_includes/type.html

```diff
@@ -0,0 +1,19 @@
+{% assign res = "" %}
```

Comment:
This file seems to be the `type` for only the "anyOf" case?   If so, rename this to `type-anyof.html` or make this file contain all logic for type?  (The former would be easier.)